packages/jellyfish-json/src/remap.ts

```diff
@@ -71,7 +114,7 @@ function remapLosslessObj (losslessObj: any, precision: PrecisionMapping): any {
         break
 
       case MappingAction.DEEPLY_UNKNOWN:
-        remapLosslessObj(losslessObj[k], {})
+        remapLosslessObj(losslessObj[k], precision)
```

Comment:
this should be `{}`

should not pass unknown mapping deeply down

Not a good idea: `{result: {a: 123} } != {a: 'bignumber'}`

Unexpected side effect of  `{foo: {a: 123}, a: 123 } != {a: 'bignumber'}` as `foo.a` should not be parsed.

Reply:
yea you're right

Reply:
I remember why I put `precision` to loss the restriction 
The ApiClient.call is currently return the resp like this `{result: {jsonvalue} , error, id}`
when doing parse we just need to put

Reply:
@fuxingloh 
here I opened issue and also for a note on current loose precision parse
hmm.. its not critical... I added the convince-able note as well 
https://github.com/DeFiCh/jellyfish/issues/104